src/cluecode/finder.py

```diff
@@ -74,6 +91,10 @@ def find(location, patterns):
 
     for i, line in enumerate(analysis.text_lines(location)):
         lineno = i + 1
+        if ['urls', 'emails'] in patterns:
```

Comment:
Why combining the two calls here? There is no reason to search for email in a url line and the same the other way, right?

Reply:
Nice catch, thanks a ton !

Reply:
Nice catch, thanks a ton !Addressed your comment. :)